src/Network/Benchmark.hs

```diff
@@ -67,9 +67,10 @@ import Control.Lens hiding ((.=))
 import Control.Monad
 import Control.Monad.Reader
 import Control.Monad.Error.Class
-import Control.Monad.Except
 import Control.Monad.Trans.Control
+import Control.Monad.Trans.Either
```

Comment:
Generally, I am slowly migrating away from `EitherT` since `ExceptT` got added to transformers, but given that `retryT` was already in the code base, I think it's fine here.